superduperdb/ext/sentence_transformers/model.py

```diff
@@ -23,6 +22,8 @@ class SentenceTransformer(_Predictor, _DeviceManaged):
     model: t.Optional[str] = None
     device: str = 'cpu'
     preprocess: t.Optional[t.Callable] = None
+    postprocess: t.Optional[t.Callable] = None
```

Comment:
we need to add a `postprocess` to help convert the array to a list, otherwise, you need to additionally process the data in numpy format during database insertion, or force conversion in the predict function below.

Reply:
Ok, good point.